doc/source/readers.rst

```diff
@@ -109,6 +109,39 @@ the :class:`~satpy.scene.Scene` initialization.
 See the :func:`~satpy.readers.find_files_and_readers` documentation for
 more information on the possible parameters.
 
+Metadata
+========
+
+The datasets held by a scene also provide vital metadata such as dataset name, units, observation time etc. The
+following attributes are standardized across all readers:
+
+* ``name``, ``wavelength``, ``resolution``, ``polarization``, ``calibration``, ``level``, ``modifiers``: See
+  :class:`satpy.dataset.DatasetID`.
+* ``start_time``: Left boundary of the time interval covered by the dataset.
+* ``end_time``: Right boundary of the time interval covered by the dataset.
+* ``orbital_parameters``: Dictionary of orbital parameters describing the satellite's position.
+
+  * ``satellite_actual_longitude/latitude/altitude``: Current position of the satellite at the time of observation in
+    geodetic coordinates (i.e. altitude is normal to the surface).
+  * For geostationary satellites there are more parameters related to the satellite position:
+
+    * ``satellite_nominal_longitude/latitude/altitude``: Centre of the station keeping box (a confined area in which
+      the satellite is actively maintained in using maneuvres). Inbetween major maneuvres, when the satellite
+      is permanently moved, the nominal position is constant.
+    * ``nadir_longitude/latitude``: Intersection of the instrument's Nadir with the surface of the earth. May differ
+      from the actual satellite position, if the instrument is poiting slightly off the axis (satellite, earth-centre).
+      If available, this should be used to compute viewing angles etc. Otherwise, use the actual satellite position.
+    * ``projection_longitude/latitude/altitude``: Projection centre of the re-projected data. This should be used to
+      compute lat/lon coordinates. Note that the projection centre can differ considerably from the actual satellite
+      position. For example MSG-1 was at times positioned at 3.4 degrees west, while the image data was re-projected
+      to 0 degrees.
+
+* [DEPRECATED] ``satellite_longitude/latitude/altitude``: Current position of the satellite at the time of observation
```

Comment:
Has this existed for enough readers and long enough to justify documenting its deprecation?

Reply:
Yes I think so. It has been introduced in 2016, and I've seen colleagues from SMHI using it.

Reply:
It occurs in eight readers

Reply:
Sounds good.